rfcs/text/0031-risk-fields.md

```diff
@@ -21,6 +21,26 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 <!--
 Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
 -->
+The `risk` fields being proposed are as follows:
+
+Field | Type | Example | Description | Use Case
+-- | -- | -- | -- | -- 
+risk.calculated_score_norm | float | 88.73 | A risk classifications core (0-100) calculates by an internal system as part of entity analytics and entity risk scoring | Can be used to indicate the risk associated with a particular host
+risk.calculated_level | keyword | High | A risk classification level calculated by an internal system as part of entity analytics and entity risk scoring | Can be used to indicate the risk associated with a particular host
+risk.static_score_norm | float | 83.0 | A risk classification score (0-100) obtained from outside the system, such as from some external Threat Intelligence Platform | Can be used to indicate the projected risk of a particular host based on a trusted third party intelligence feed 
+risk.static_level | keyword | High | A risk classification level obtained from outside the system, such as from some external Threat Intelligence Platform | Can be used to indicate the projected risk of a particular host based on a trusted third party intelligence feed
+risk.factors| object | See Source data section | Factors that contributed to the risk | Explainability about what contributed to the risk
+risk.factors.alerts | array of objects (Optional) | See Source data section | Alerts that contributed to the risk | Explainability about what contributed to the risk
+risk.factors.alerts.rule_id | keyword | 99cb72b7-ab55-4375-8bbe-696e3e1b8fe2 | Rule ID associated with the alert | Metadata about alerts that contributed to the risk
+risk.factors.alerts.rule_name | keyword | Unusual Windows Path Activity | Rule name associated with the alert | Metadata about alerts that contributed to the risk
+risk.factors.alerts.rule_risk | float | 2.41 | Risk associated with the alert | Metadata about alerts that contributed to the risk
+risk.factors.others | array of strings (Optional) | See Source data section | Factors apart from alerts that contributed to the risk | Explainability about what contributed to the risk
```

Comment:
If 'risk.factors.others' has both a 'type' and a 'description', should 'others' be type object array instead of string array?